cypress/integration/expert.spec.js

```diff
@@ -2,7 +2,8 @@
 
 describe("Community members", () => {
     beforeEach(() => {
-        cy.login("community", "1234567890");
+        cy.login("expert", "1234567890");
+        cy.get('#djHideToolBarButton').click();
```

Comment:
In a future PR, it might be useful to create a new [Cypress custom command](https://docs.cypress.io/api/cypress-api/custom-commands) to hide the debug toolbar.